plugins/kosync.koplugin/main.lua

```diff
@@ -300,6 +307,29 @@ If set to 0, updating progress based on page turns will be disabled.]]),
                     UIManager:show(items)
                 end,
             },
+            {
+                text = _("Document matching method"),
+                sub_item_table = {
+                    {
+                        text = _("Binary. Only identical files will sync progress."),
+                        checked_func = function()
+                            return self.kosync_checksum_method == CHECKSUM_METHOD.BINARY
+                        end,
+                        callback = function()
+                            self:setChecksumMethod(CHECKSUM_METHOD.BINARY)
+                        end,
+                    },
+                    {
+                        text = _("File-name. Files with the same file name will sync progress."),
```

Comment:
Nit: `File name` or `Filename`

Reply:
Let's go with filename.
```suggestion
                        text = _("Filename. Files with the same file name will sync progress."),
```